src/common/buffer.cc

```diff
@@ -165,8 +166,10 @@ static ceph::spinlock debug_lock;
     MEMPOOL_CLASS_HELPERS();
 
     raw_posix_aligned(unsigned l, unsigned _align) : raw(l) {
+      // posix_memalign() requires a multiple of sizeof(void *)
+      if (_align < sizeof(void *))
+	_align = sizeof(void *);
```

Comment:
nit: how about replacing this conditional with `std::max(align, sizeof(void*))` directly on the initializer list?

Reply:
I don't like putting logic in the initializer list.  I did it in the constructor body instead.